staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -31,6 +34,52 @@ type DelayingInterface interface {
 	Interface
```

Comment:
I suggest a comment to this explaining the "active" queue is things we will process asap (just need a free worker) and the "delaying" queue -- implemented by this type -- is things that are waiting for time to pass.

Reply:
Have done